go.mod

```diff
@@ -22,6 +22,6 @@ module github.com/apache/incubator-yunikorn-scheduler-interface
 go 1.12
 
 require (
-	github.com/golang/protobuf v1.3.1
-	google.golang.org/grpc v1.21.1
+	github.com/golang/protobuf v1.3.2
```

Comment:
In the Makefile we clone and checkout protobuf 1.3.1.
Changing the version here is only half of what should be done. We really should pass the version from the `go.mod` file via the root Makefile into the `lib/go` Makefile.

Reply:
Thanks @wilfred-s . I think it makes sense to keep them in sync.
I have modified the Makefile accordingly, to make sure the protobuf version is consistent with what's been defined in the `go.mod`. Please help to review.